Tasks/AzureFunctionAppV1/task.json

```diff
@@ -17,7 +17,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 1,
-    "Minor": 231,
+    "Minor": 236,
```

Comment:
Resolved and included the build from Common packages for 236.

Reply:
@v-mohithgc  @DergachevE @kirill-ivlev - sprint validation logic is here:  https://github.com/microsoft/azure-pipelines-tasks/blob/master/ci/check-downgrading.js -- if we have specific requirements which isn't being caught by the validation, it would be better to update the validation

Reply:
@v-mohithgc  I unblocked the 'waiting' state, I hope you don't mind.